<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabVsClass</class>
 <widget class="QMainWindow" name="LabVsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1278</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LabVs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ImageCropWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImages1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNextImage1">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>230</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTransform">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCrop">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sectorSpinBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>350</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sectors:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSegm">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonShowData">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>ShowAllData</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImages2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImages3">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTransform_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTransform_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTransform_All">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>TransformAll</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>550</y>
      <width>301</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>550</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSelectImages4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>30</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load 4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonTransform_4">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>230</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSegm_2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>380</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Segmentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCompare">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Compare</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCompare_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>440</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>CompareD</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageCropWidget</class>
   <extends>QWidget</extends>
   <header location="global">imagecropwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="LabVs.qrc"/>
 </resources>
 <connections/>
</ui>
